pkg/cli/internal/initialize/init.go

```diff
@@ -27,12 +27,11 @@ var errNoWriter = errors.New("configWriter is required")
 //go:embed conduit.yml.example
 var sampleConfig string
 
-func formatArrayObject(obj string) string {
-
+func formatObject(obj string, isArray bool) string {
```

Comment:
The main changes is to use this function for each type of configuration. Once that was done, I updated each of the sample files to use consistent padding.